### DISCUSSION TITLE
when i use apisix connect keycloak ,it return error Invalid bearer token,but actually the token is correct

### DISCUSSION THREAD

**Discussion by Coderyangg** [apache/apisix#9102] [Q&A]
here is my post request ,it is used to get uma token,when i post request to keycloak ,it return me a uam token
![image](https://user-images.githubusercontent.com/39979426/225843255-eee72afe-25b2-44ec-b5b9-3df281b732dc.png)
but when i use apisix,it return like this
![image](https://user-images.githubusercontent.com/39979426/225843484-417a48d6-b6a7-4347-a4c5-91033139ed19.png)
this is my apisix settings 
![image](https://user-images.githubusercontent.com/39979426/225843619-767d1f06-e96b-4ab8-917d-44888e6bcf7e.png)

**Comment by woyouyitouchenfeijv** [comment]
401 ，Is this the return value of your interface
